graphgym/run_batch.sh

```diff
@@ -1,11 +1,11 @@
 #!/usr/bin/env bash
 
-CONFIG=example_node
-GRID=example
-REPEAT=3
-MAX_JOBS=8
-SLEEP=1
-MAIN=main
+CONFIG=${CONFIG:="example_node"}
+GRID=${GRID:="example"}
```

Comment:
```suggestion
GRID=${GRID:-example}
```
No need for the quality here as you are setting it anyway, the equality is used to do an assignment, see https://unix.stackexchange.com/questions/122845/using-a-b-for-variable-assignment-in-scripts
Also you don't need the quotes?